docs/reference/search/rrf.asciidoc

```diff
@@ -110,9 +112,58 @@ RRF does not currently support:
 * <<request-body-search-explain, explain>>
 * <<profiling-queries, profiling>>
 
-Using unsupported features as part of a search using RRF will result
+Using unsupported features as part of a search using RRF results
 in an exception.
 
+[[rrf-using-sub-searches]]
+==== Reciprocal rank fusion using sub searches
+
+<<request-body-sub-searches, Sub searches>> provides a way to
+combine and rank multiple searches using RRF.
+
+An example request using RRF with sub searches:
+
+[source,console]
+----
+GET example-index/_search
+{
+    "sub_searches": [
+        {
+            "query": {
+                "term": {
+                    "text": "blue shoes"
+                }
+            }
+        },
+        {
+            "query": {
+                "term": {
+                    "text": "red shoes"
+                }
+            }
+        }
```

Comment:
what is the end goal of this request? Should we provide an example that is closer to how we expect users to use this feature?

Reply:
Yes. I will try re-work this with an ELSER query instead.

Reply:
I mentioned the same thing in my review, but just noticed it was still `pending` and I didn't hit submit 🤦 .

The short of it, I agree with @javanna , we should use `text_expansion` or `script_query`. Something that scores in a different distribution than BM25

Reply:
Updated to use a text expansion query.